Okta.Core.Automation/Users/DeleteOktaUser.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Management.Automation;
+using Okta.Core;
+using Okta.Core.Models;
+
+namespace Okta.Core.Automation
+{
+    [Cmdlet("Delete", "OktaUser")]
+    public class DeleteOktaUser : OktaCmdlet
+    {
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "ID or username of the user to delete"
+        )]
+        public string Id { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 1,
+            HelpMessage = "Search query string (see http://developer.okta.com/docs/api/resources/users.html#find-users)"
+        )]
+        public string Query { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 2,
+            HelpMessage = "Use a filter (see http://developer.okta.com/docs/api/resources/users.html#list-users-with-a-filter)"
+        )]
+        public string Filter { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
+            HelpMessage = "Use advanced search (see http://developer.okta.com/docs/api/resources/users.html#list-users-with-search)"
+        )]
+        public bool Search { get; set; }
+
+        protected override void ProcessRecord()
+        {
+            var usersClient = Client.GetUsersClient();
+            try
+            {
+                List<User> users = new List<User>();
+                if (!string.IsNullOrEmpty(Id))
+                {
+                    var user = usersClient.Get(Id);
+                    if (user != null)
```

Comment:
Fixed in [separate commit](https://github.com/okta/oktasdk-csharp/pull/18/commits/77dbc421811bd23e77cf4417bb443ff0d370a070).